app/dependencies.py

```diff
@@ -32,8 +35,13 @@ def __call__(self, session: DBSession) -> handlers.ModelHandler:
     handlers.SystemHandler, Depends(HandlerFactory(handlers.SystemHandler))
 ]
 
+
+def get_account_handler(session: DBSession) -> AccountHandler:
+    return AccountHandler(session)
+
+
```

Comment:
this function is not needed anymore, you are using the `HandlerFactory` at line #27.